### DISCUSSION TITLE
Welcome to GeminiProChat Discussions!

### DISCUSSION THREAD

**Discussion by babaohuang** [babaohuang/GeminiProChat#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bamudx32** [comment]
gemini 可以支撑图片视频上传 能不能在这里面也对应开发一下呀 大佬

**Comment by lovese** [comment]
提示无法联网查询😂

**Comment by leealu** [comment]
这个返回日期是 2021 年 4月以前的

**Comment by TazmiDev** [comment]
This situation occurs when using your own domain name after deploying with Vercel, but you can access it normally under a VPN environment. Is there a way to solve this problem without using a VPN?
![daac1228154119](https://github.com/babaohuang/GeminiProChat/assets/152580586/dfcdcd96-5c77-4436-bf4a-3a6529828296)

**Reply by TazmiDev** [reply]
use proxy.
